include/rcutils/get_env.h

```diff
@@ -61,6 +61,30 @@ RCUTILS_WARN_UNUSED
 const char *
 rcutils_get_env(const char * env_name, const char ** env_value);
 
+// Retrieve the full path to the home directory.
+/*
+ * The c-string which is returned is only valid until the next time this
+ * function is called, because it is a direct pointer to the static storage.
+ * Also note that the string returned here should *not* be freed.
+ *
+ * The function work by first trying to get the contents of the HOME environment
```

Comment:
```suggestion
 * The function works by first trying to get the contents of the HOME environment
```

Remember suggestions mess with DCO if you don't include the sign-off in the commit message.

Reply:
I've revamped the language in 3352d19 so this doesn't directly apply anymore; please take a look.